multiple_generate.sh

```diff
@@ -7,19 +7,18 @@ function clean() {
 function cloner() {
   URL_SPLIT=(${1//// })
   REPONAME=(${URL_SPLIT[3]//./ })
-  git clone $1 ${REPONAME}_temp &
+  git clone $1 ${REPONAME} &
 }
 
-function repoCleaner() {
+function genSubIndex() {
   URL_SPLIT=(${1//// })
   REPONAME=(${URL_SPLIT[3]//./ })
-  cd ${REPONAME}_temp
-  mkdir ../$REPONAME
-  cp -a docs/. ../$REPONAME/.
-  cd ..
-  rm -rf ${REPONAME}_temp
+  cd $REPONAME/docs
+  python $BASE/modules/scripts/genindex.py ./
```

Comment:
I would advise you to once again go through `genindex.py`.
Basically, there are two directories, docpath and the root directory which contains README.
The script is run from `docpath`, so we have access to docpath by os.curdir. And the script takes a parameter `root_dir` so that it has access to the directory containing README. Makes sense?